File: app/src/main/java/com/dss886/transmis/utils/Logger.java
package com.dss886.transmis.utils;

import android.util.Log;
import com.dss886.transmis.BuildConfig;

/**
 * Created by dss886 on 2017/6/29.
 */

public class Logger {

    // TODO: 2021/02/11 @duansishu add Tag argument
    public static void d(String message) {
        if (BuildConfig.DEBUG) {
            Log.d("Transmis", message);
        }
    }

    public static void e(String message) {
        if (BuildConfig.DEBUG) {
            Log.e("Transmis", message);
        }
    }
}

File: app/src/main/java/com/dss886/transmis/utils/Logger.kt
package com.dss886.transmis.utils

import android.util.Log
import com.dss886.transmis.BuildConfig

/**
 * Created by dss886 on 2017/6/29.
 */
object Logger {

    fun d(tag: String, message: String) {
        if (BuildConfig.DEBUG) {
            Log.d(tag, message)
        }
    }

    fun e(tag: String, message: String) {
        if (BuildConfig.DEBUG) {
            Log.e(tag, message)
        }
    }
}